Makefile

```diff
@@ -32,6 +32,10 @@ BUILDER_IMAGE ?= golang:1.17
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.23
 
+# If this change will be impactful to you please leave a comment on https://github.com/onsi/ginkgo/issues/711
+# Learn more at: https://github.com/onsi/ginkgo/blob/ver2/docs/MIGRATING_TO_V2.md#removed-custom-reporters
+# To silence deprecations that can be silenced set the following environment variable:
+ACK_GINKGO_DEPRECATIONS=1.16.5
```

Comment:
Can we move to v2 without controller-runtime needing to do so as well?

Please open an issue and put a TODO here to move to v2.

Reply:
https://github.com/kubernetes-sigs/kueue/issues/94

Reply:
Put `TODO(#92): upgrade ginkgo` in the comment.